data/wanderers middle.txt

```diff
@@ -1766,7 +1766,82 @@ mission "Wanderers: Sestor Search: Varu K'prai"
 			`	The Wanderers don't know what happened next, but the drones are no longer here, so they must have moved on to a different system.`
 				decline
 
-
```

Comment:
Keep three blank lines between missions for the sake of readability, please (so if scanning quickly through the file, it's easy to see where each mission definition begins and ends).